ffi/examples/read-table/schema.h

```diff
@@ -90,7 +96,7 @@ void print_physical_name(const char *name, const CStringMap* metadata)
 #ifdef VERBOSE
   char* key_str = "delta.columnMapping.physicalName";
   KernelStringSlice key = { key_str, strlen(key_str) };
-  char* value = get_from_map(metadata, key, allocate_string);
+  char* value = get_from_string_map(metadata, key, allocate_string);
```

Comment:
this was just wrong before. we actually couldn't compile with `VERBOSE` set to true